### DISCUSSION TITLE
你的下一款邦布 何必是邦布

### DISCUSSION THREAD

**Discussion by GrunderIndustries** [share121/inter-knot#4606] [科技]
![image](https://github.com/user-attachments/assets/dfbbcc91-781e-4208-a779-a2263cb5814c)
# 你还在为邦布差劲火力不足而烦恼吗!
# 你还在为拿不下空洞制空权而困扰吗!
# 你还在为只能看着飞行以骸 在你头上拉屎却无能为力而愤怒吗!
## 现在订购 创立者工业研发的
# MQ-99吧!
## 无论是拦截飞艇,精准打击,还是把地上的以骸打成碎片
# 它都能做到！
# 它体积小巧 相当轻便 满载也仅仅重250kg 可以轻松装入卡车进行运输
## 其可以装备AGM-179空地导弹 GBU-53B激光制导炸弹的情况下
# 还能装备两枚AIM-9 和一门20mm机炮！
## 还搭载了邦布AI转移 空洞内导航 远程遥控功能！
# 仅售120000丁尼

*创立者工业不对潜在的的法律风险和人身安全风险负责
*不附赠除机炮外的武器系统

**Comment by tsubaki903726** [comment]
这个委托我接了

**Reply by yang12535** [reply]
byd碰到脚本了

**Comment by sensei-is-playing-GenShin** [comment]
我是学生，送我

**Comment by Orangered81** [comment]
可恶，都逃到这里了还有月计（

**Comment by aomengxin** [comment]
购物委托你接了

**Comment by AnEwei** [comment]
我嘞个巴巴托斯阿

**Comment by Rae-CY** [comment]
有买家展示图吗？

**Comment by tiaozi999** [comment]
才12w？我吃个面就花了5000，你这邦布行不行啊？
